File: i2f-jdk/i2f-cache/src/main/java/i2f/cache/base/ICache.java
package i2f.cache.base;

/**
 * @author Ice2Faith
 * @date 2024/6/25 13:58
 * @desc
 */
public interface ICache<K, V> {
    V get(K key);

    void set(K key, V value);

    boolean exists(K key);

    void remove(K key);
}

File: i2f-jdk/i2f-swl/web/i2f-swl/src/i2f-turbo-web/i2f-core/cache/ICache.js
/**
 * @retunrn {ICache}
 * @interface
 */
function ICache() {

}

/**
 *
 * @param key {String}
 * @return {Object}
 */
ICache.prototype.get = function (key) {

}
/**
 *
 * @param key {String}
 * @param value {Object}
 * @return {void}
 */
ICache.prototype.set = function (key, value) {

}

/**
 *
 * @param key {String}
 * @return {boolean}
 */
ICache.prototype.exists = function (key) {

}

/**
 * @param key {String}
 * @return {void}
 */
ICache.prototype.remove = function (key) {

}

export default ICache
